### PR TITLE
Harmonize FLOSS blocks docs

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Overall PR: https://github.com/up42/snap-polarimetric/pull/9
Tracked by https://geoinformationstore.atlassian.net/browse/UP-3132

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,185 +1,171 @@
...
-set it to some other directory that is convenient to you.
 
-### Publish the block to UP42
 
-#### Authenticate into the UP42 registry 
+## Pushing the block to the UP42 platform
 
-Login into the UP42 [Docker image registry](https://docs.docker.com/registry/) 
-with your UP42 user ID (`<user_id>`) and password:
+First login to he UP42 docker registry. `me@example.com` needs to be replaced by your **UP42 username**,
```
Comment (nekhtiari) [inline README.md]: Typo -> First login to the

**Comment by nekhtiari** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,185 +1,171 @@
...
-easily readable by GIS packages such as QGIS (https://www.qgis.org/).
-You need to create a `/tmp/output/` folder so that the output image can be written into this directory.
-If you want to also copy the original high resolution (10m bands) you can do so, with tuning the parameter
-`copy_original_bands` in the Up42Manifest.json and choose default to be `yes`.
-The source code also predict the lowest resolution bands (60m) by default, so that the output image will include high
-resolution (10m) for all the exiting bands within 20m and 60m resolutions.
+## Testing the block locally
 
+Before uploading the block to the UP42 platform, we encourage you to run the following local tests and validations to
+ensure that the block works as expected, conforms to the UP42 specifications and can could successfully applied in an
```
Comment (chrieke) [inline README.md]:
```suggestion
ensure that the block works as expected, conforms to the UP42 specifications and could be successfully applied in an
```

File: README.md (hunk 1 above)
Comment (rodrigoalmeida94) [reply README.md]: Good catch!

File: README.md (hunk 2 above)
Comment (rodrigoalmeida94) [reply README.md]: Thanks!

**Pull request merged**